lua/lspconfig/jdtls.lua

```diff
@@ -141,7 +141,7 @@ inferred. Please set the following environmental variables to match your install
 ```bash
 export JAR=/path/to/eclipse.jdt.ls/org.eclipse.jdt.ls.product/target/repository/plugins/org.eclipse.equinox.launcher_1.6.0.v20200915-1508.jar
 export GRADLE_HOME=$HOME/gradle
-export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-11.0.9.11-9.fc33.x86_64/
+export JAVA_HOME=$(readlink -f /usr/bin/java | sed "s:/bin/java::")
```

Comment:
Unfortunately this isn's as portable as you think it is. It only works on GNU Linux. The `-f` flag doesn't exist on the BSD variant (e.g. macOS).

Seems like the post portable solution might be `realpath(1)` or Perl/Python one-liner. Though I guess they also have some caveats.

```bash
perl -MCwd -le 'print Cwd::abs_path shift' /usr/bin/java
```
```bash
python3 -c 'import os, sys; print(os.path.realpath(sys.argv[1]))' /usr/bin/java
```
```bash
realpath /usr/bin/java
```

https://stackoverflow.com/questions/1055671/how-can-i-get-the-behavior-of-gnus-readlink-f-on-a-mac